### PR TITLE
Tighten up Content-Type checking

### PR DESCRIPTION
**Pull request by gaul**

### CODE REVIEW

File: test/test-utils.sh [hunk 1]
```diff
@@ -315,11 +315,11 @@ function get_permissions() {
 
 function check_content_type() {
     local INFO_STR
-    INFO_STR=$(aws_cli s3api head-object --bucket "${TEST_BUCKET_1}" --key "$1")
-    if [[ "${INFO_STR}" != *"$2"* ]]
+    INFO_STR=$(aws_cli s3api head-object --bucket "${TEST_BUCKET_1}" --key "$1" | jq -r .ContentType)
+    if [[ "${INFO_STR}" != "$2" ]]
```
Comment (ggtakec) [inline test/test-utils.sh:319]: I think the double brackets are no longer needed.

**Review by ggtakec — COMMENTED** [review]
Please check one comment.

File: test/test-utils.sh (hunk 1 above)
Comment (gaul) [reply test/test-utils.sh:319]: Done.

Comment (ggtakec) [reply test/test-utils.sh]: Thanks.

**Review by ggtakec — APPROVED** [review]

**Pull request merged**